coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/subaward/BudgetSubAwards.java

```diff
@@ -114,9 +113,6 @@
     @OneToMany(mappedBy = "budgetSubAward", orphanRemoval = true, cascade = { CascadeType.ALL })
     @OrderBy("budgetPeriod")
     private List<BudgetSubAwardPeriodDetail> budgetSubAwardPeriodDetails;
-
-    @OneToMany(mappedBy = "budgetSubAward", cascade = {CascadeType.REFRESH, CascadeType.REMOVE})
-    private List<BudgetLineItem> budgetLineItems;
     
```

Comment:
So previously we were mapping but not managing this relationship. So when new line items were added they weren't added here and attempting to add them here when creating them caused a problem because we don't want to persist this relationship since the BudgetPeriod technically owns the line items and JPA throws an exception in this case. And since SubAward doesn't actually own these child objects, I don't think orpahanRemoval is technically correct either so this mapping is unusual and complicated.

So we can keep this relationship and then have to call refreshReference on it whenever we need to use it which I hate and isn't really correct. We should strive to keep the object graph correct with the relationships. And I don't see how to do that in this case and I figured removing the relationship the most correct course.